src/main/kotlin/racing/domain/Cars.kt

```diff
@@ -0,0 +1,22 @@
+package racing.domain
+
+class Cars(count: Int) : Iterable<Car> {
+
+    private val list: List<Car>
+
+    init {
+        list = List(count) {
+            Car()
+        }
+    }
+
+    fun count() = list.size
+
+    fun race(movable: () -> Int) {
+        list.forEach {
+            car ->
+            car.move(movable.invoke())
+        }
```

Comment:
```suggestion
        list.forEach { car ->
            car.move(movable.invoke())
        }
```
요런 포멧을 많이 사용하는 것 같아요. 

+) `it` 키워드를 활용해도 좋을 것 같습니다.

Reply:
넵!